.lintr

```diff
@@ -1,9 +1,10 @@
-linters:
-  linters_with_defaults(
-    line_length_linter = line_length_linter(100)
-  )
-exclusions:
-  c(
-    "inst/rstudio",
-    "inst/templates"
-  )
+linters:
+  linters_with_defaults(
+    line_length_linter = line_length_linter(100)
+  )
+exclusions:
+  c(
+    "inst/rstudio",
+    "inst/templates",
+    "tests/e2e/app-files"

```

Comment:
I had to exclude the `app-files` because there are `box` calls here that trigger object usage linters.